app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -97,6 +98,10 @@ open class HomeActivity : AppCompatActivity() {
 
         setupToolbarAndNavigation()
 
+        if (intent.getBooleanExtra(EXTRA_FINISH_ONBOARDING, false)) {
```

Comment:
nit: Not your fault but I don't think `FenixOnboarding` is an intuitive name: it should be something like, `UserOnboardedState`. Similarly, I don't think `finish` is an intuitive name for that method either. It's more of a, `setUserOnboarded`. Relatedly, I find `EXTRA_FINISH_ONBOARDING` to be misleading too. I don't think this is worth the time to fix now.

On that note, if you happen upon code that you think could be better and relatively easy to change, I think it's okay for us to make those sorts of changes too. :)